functions-and-scope.js

```diff
@@ -14,6 +17,49 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
 
+// Met een arrow function en return value
+const cumLaude = () => {
+    let cumLaude = null
+    for (let i = 0; i < grades.length; i++) {
+        if (grades[i] >= 8) {
+            cumLaude ++
+        }
+    }
+    return cumLaude
+}
+console.log(cumLaude())
+
+// Normale function met console log inplaats van return value
+function cumLaudeA() {
+    let counter = null
+    for (let i = 0; i < grades.length; i++) {
+        if (grades[i] >= 8) {
+            counter ++
+        }
+    }
+    console.log(counter)
+}
+cumLaudeA()
+
+// Met een nieuwe array en felicitaties
+
+const OnlyCumLaudeArray = []
```

Comment:
Variabelen camelCase schrijven, net als functienamen
```suggestion
const onlyCumLaudeArray = []
```